provider/cmd/pulumi-gen-aws-native/main.go

```diff
@@ -73,68 +69,57 @@ func main() {
 		return
 	}
 
-	supportedTypes := readSupportedResourceTypes(genDir)
-	jsonSchemas := readJsonSchemas(schemaFolder)
-	semanticsDocument, err := schema.GatherSemantics(semanticsDir)
-	if err != nil {
-		panic(fmt.Sprintf("error gathering semantics: %v", err))
-	}
-	fullSpec, _, _, err := schema.GatherPackage(supportedTypes, jsonSchemas, true, &semanticsDocument)
-	if err != nil {
-		panic(fmt.Sprintf("error generating schema: %v", err))
-	}
-	fullSpec.Version = version
-
 	for _, language := range strings.Split(languages, ",") {
```

Comment:
Why do we still need this loop if the only remaining language is "schema"? Worth simplifying?